projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -691,7 +691,7 @@ class="screen-reader-text"
 							style="<?php echo esc_attr( $styles['submit_button_wrapper'] ); ?>"
 						<?php endif; ?>
 					>
-						<input type="hidden" name="action" value="subscribe"/>
+						<input type="hidden" name="action" value="<?php echo is_top_subscription() ? 'subscribed' : 'subscribe'; ?>"/>
```

Comment:
You could also add to `<form>` as a `data-` prop and then query with `form.dataset.action` in JS. Feels simpler.